lib/tests/release.nix

```diff
@@ -1,40 +1,32 @@
-{ nixpkgs ? { outPath = (import ../.).cleanSource ../..; revCount = 1234; shortRev = "abcdef"; }
-, # The platforms for which we build Nixpkgs.
-  supportedSystems ? [ builtins.currentSystem ]
-, # Strip most of attributes when evaluating to spare memory usage
-  scrubJobs ? true
-}:
+{ pkgs ? import ((import ../../lib).cleanSource ../..) {} }:
```

Comment:
You will have to update the list of attribute given when this expression is imported in `pkgs/top-level/release.nix` to make this evaluate correctly.